File: Practice/Language-Specific/C/HelloWorld.c
/*
	HelloWorld.java by Randyll Bearer
	Simple Program to test development environment and C/GCC install
*/

#include <stdlib.h>
#include <stdio.h>
#include <string.h>

char* getPhrase(){
	
	char *message = malloc(sizeof(char) * 14);
	strcpy(message, "Hello, World!");
	return message;
	
}

int printPhrase(char *phrase){
	
	printf(phrase);
	
	return 1;
	
}

int main(){
	
	char *phrase = getPhrase();
	
	if(printPhrase(phrase) == 1 ){
		free(phrase);	//always free after malloc
		return 0;
	}else{
		printf("error");
		return 1;
	}
	
}
//End of File

File: Practice/Language-Specific/Java/HelloWorld.java
/*
	HelloWorld.java by Randyll Bearer
	Simple Program to test development environment and Java install
*/


public class HelloWorld {

	public static void main(String args[] ){
		
		String phrase = "";
		phrase = getPhraseToPrint();
		printToScreen(phrase);
		
		System.exit(0);	//Make sure our program ends
		
	}

	private static void printToScreen(String toPrint){
		
		System.out.println(toPrint);
	
	}
	
	
	
	private static String getPhraseToPrint(){
		return "Hello World!";
	}

} 
//End of File